server/xpub-controller/notification.js

```diff
@@ -0,0 +1,82 @@
+const Joi = require('joi')
+const pug = require('pug')
+const dns = require('dns')
+const util = require('util')
+const mailer = require('@pubsweet/component-send-email')
+const logger = require('@pubsweet/logger')
+
+class Notification {
+  constructor(config, people) {
+    this.resolveMx = util.promisify(dns.resolveMx)
+    this.people = people
+    this.config = config
+  }
+
+  async isValidEmail(email) {
+    let result = false
+    try {
+      await Joi.validate(
+        email,
+        Joi.string()
+          .email()
+          .required(),
+      )
+      const tld = email.split('@')[1]
+      const lookup = await this.resolveMx(tld)
+      result = lookup.length > 0
+    } catch (err) {
+      return false
+    }
+
+    return result
+  }
+
+  async sendDashboardEmail() {
+    const emailList = this.people.map(person => person.alias.email).join(',')
+    const valid = await this.people
+      .map(async person => this.isValidEmail(person.alias.email))
+      .reduce((previous, current) => previous && current)
+
+    if (!valid) {
+      return false
```

Comment:
It's worth logging here, if only so that we can manually test it in the review environment.